src/main/java/uk/ac/ebi/eva/pipeline/jobs/CommonJobStepInitialization.java

```diff
@@ -11,6 +26,11 @@
 
 import uk.ac.ebi.eva.pipeline.configuration.VariantJobsArgs;
 
+/**
+ * Helper class to build jobs.
+ *
+ * This is not intended to be used as an actual job, but to be extended in those actual job classes.
```

Comment:
If it's not intended to a job, shouldn't we make it abstract? @jorizci, is there any reason not to?